tests/integration/cartography/intel/aws/test_secretsmanager.py

```diff
@@ -63,3 +66,105 @@ def test_load_load_secrets(neo4j_session, *args):
         for n in nodes
     }
     assert actual_nodes == expected_nodes
+
+
+def test_load_secret_versions(neo4j_session, *args):
+    """
+    Ensure that expected secret versions get loaded with their key fields.
+    """
+
+    secret_data = [
+        {
+            "ARN": "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-1-000000",
+            "Name": "test-secret-1",
+            "Description": "This is a test secret",
+            "RotationEnabled": True,
+            "RotationRules": {"AutomaticallyAfterDays": 90},
+            "RotationLambdaARN": "arn:aws:lambda:us-east-1:000000000000:function:test-secret-rotate",
+            "KmsKeyId": "arn:aws:kms:us-east-1:000000000000:key/00000000-0000-0000-0000-000000000000",
+            "CreatedDate": datetime.datetime(2014, 4, 16, 18, 14, 49, tzinfo=tz.utc),
+            "LastRotatedDate": datetime.datetime(
+                2014, 4, 16, 18, 14, 49, tzinfo=tz.utc
+            ),
+            "LastChangedDate": datetime.datetime(
+                2014, 4, 16, 18, 14, 49, tzinfo=tz.utc
+            ),
+            "LastAccessedDate": datetime.datetime(
+                2014, 4, 16, 18, 14, 49, tzinfo=tz.utc
+            ),
+            "PrimaryRegion": "us-west-1",
+        }
+    ]
+    cartography.intel.aws.secretsmanager.load_secrets(
+        neo4j_session,
+        secret_data,
+        TEST_REGION,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    data = tests.data.aws.secretsmanager.LIST_SECRET_VERSIONS
+    cartography.intel.aws.secretsmanager.load_secret_versions(
+        neo4j_session,
+        data,
+        TEST_REGION,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    expected_nodes = {
+        (
+            "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-1-000000:version:00000000-0000-0000-0000-000000000000",
+            "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-1-000000",
+            "00000000-0000-0000-0000-000000000000",
+            ("AWSCURRENT",),
+            "us-east-1",
+            1397672089,
+        ),
+        (
+            "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-1-000000:version:11111111-1111-1111-1111-111111111111",
+            "arn:aws:secretsmanager:us-east-1:000000000000:secret:test-secret-1-000000",
+            "11111111-1111-1111-1111-111111111111",
+            ("AWSPREVIOUS",),
+            "us-east-1",
+            1397585689,
+        ),
+    }
+
+    nodes = neo4j_session.run(
```

Comment:
Use `check_rels()` and `check_nodes()` instead. IMO `check_rels()` is more important.